.github/workflows/ci-quality-gates.yml

```
@@ -318,126 +344,12 @@ jobs:
-          echo "Baseline: ${BASE:-<none — no reachable previous tip, nothing to enforce>}"
-
-      - name: Changelog immutability guard
-        if: steps.should_run.outputs.run == 'true' && matrix.check == 'database-migrations' && steps.baseline.outputs.base != ''
+      - name: Evaluate API and database contracts
+        if: steps.should_run.outputs.run == 'true' && matrix.check == 'server-contracts' && always()
         env:
-          BASE: ${{ steps.baseline.outputs.base }}
-          RELEASED_REF: ${{ github.event_name == 'pull_request' && github.base_ref || 'main' }}
+          OPENAPI: ${{ steps.openapi.outcome }}
+          DATABASE: ${{ steps.database.outcome }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1046:error:1:1: Couldn't find 'fi' for this 'if'

(shellcheck)